### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Barna-Molnar**
I have already done the tests we have spoken about and this is a pull request to you to review them before weekend if happen to be that you have time

### CODE REVIEW

**Review by sleepingevil — CHANGES_REQUESTED** [review]
You should not commit .eslintcache and package-lock.json

File: public/index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 
 <head>
-    <meta charset="utf-8" />
```
Comment (sleepingevil) [inline public/index.html]: Why did you remove this?

File: src/Account.jsx [hunk 2]
```diff
@@ -23,7 +23,7 @@ class Account extends React.Component {
       repayAmount: "",
       repayMessage: "",
       repayRecepient: "",
-      time: 300,
+      time: 3000,
```
Comment (sleepingevil) [inline src/Account.jsx]: What is "time" used for? You should come up with a more descriptive name for this. Does it even belong in the state?

File: src/App.jsx [hunk 3]
```diff
@@ -4,6 +4,7 @@ import Account from "./Account.jsx";
 import Overlay from "./Overlay.jsx";
 import Login from "./Login.jsx";
 import React from "react";
+import NewOperator from "./NewOperator.jsx";
```
Comment (sleepingevil) [inline src/App.jsx]: Where are you using NewOperator?

File: src/App.scss [hunk 4]
```diff
@@ -63,14 +63,9 @@
 .login-transition-enter-active {
     opacity: 1;
     transform: translateX(0%);
-    // transition: opacity 800ms, transform 800ms ease-in-out;
     transition: all 800ms ease-in-out 0.25s;
 }
 
-// .login-transition-enter-done {
```
Comment (sleepingevil) [inline src/App.scss:70]: Nice job on the clean-up. 👍 Keep your codebase clean, it will save you from a lot of headache.

File: src/mixins.scss [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    }
+    @if $breakpoint==big-desktop {
+        @media only screen and (min-width: 112.5em) {
+            @content
+        }
+        ; //1800px
+    }
+}
+
+// @include respond(tab-land) {
```
Comment (sleepingevil) [inline src/mixins.scss:34]: 👎

File: src/updateAccsRepay.js [hunk 6]
```diff
@@ -24,7 +24,7 @@ export const updateAccsRepay = (fromAcc, forAcc, transactionTyp, amount, message
                 movements: [{
                         amount: Number(amount),
                         date: date,
-                        transactionTyp: "transactionTyp",
+                        transactionTyp: transactionTyp,
```
Comment (sleepingevil) [inline src/updateAccsRepay.js:29]: LoL 🤣

File: src/updateFunctions.test.js [hunk 7]
```diff
@@ -0,0 +1,561 @@
+import { updateData, addMovement, findDeleteOrDecrease, updateCurrAcc, updateAccsLend } from './updateFunctions';
+import { accounts, date } from './data';
+import { updateAccsRepay } from './updateAccsRepay';
+
+/// 24.02.2021 Working on tests
+////////// Testing updateAccsRepay()
+
+describe("updateAccsRepay()", () => {
+    const acc1 = {
+        owner: 'Acc1',
```
Comment (sleepingevil) [inline src/updateFunctions.test.js]: Tip of the day: If you refactor your tests, and move test data generation into functions, you can make your test files shorter. Happy to show you what I mean later.

File: src/updateFunctions.test.js [hunk 8]
```diff
@@ -0,0 +1,561 @@
...
+    let state = { accounts }
+
+
+    test(`update Accounts by repayment`, () => {
+        const result = updateAccsRepay("a2", "a1", "repayment", 2000, "nothing", state, prev, date)
+
+        //Assertion 
+
+        expect(result).toEqual([{
+            owner: 'Acc1',
```
Comment (sleepingevil) [inline src/updateFunctions.test.js]: Nicely done 👍 Next level: How can you make this shorter? (Hint: There are many different possible ways)

File: src/updateFunctions.test.js [hunk 9]
```diff
@@ -0,0 +1,561 @@
...
+                sender: 'a3',
+                recepient: 'a1',
+                message: `i don't know`
+            }],
+            debt: [],
+            lended: [{ to: "a2", value: 1000 }, { to: "a1", value: 1000 }],
+        }]);
+    });
+
+    test(`update accs by lend when acc does exists in lended array`, () => {
```
Comment (sleepingevil) [inline src/updateFunctions.test.js]: Fairly good test names. 👍  The word "update" is too generic, though.

File: src/updateFunctions.test.js [hunk 10]
```diff
@@ -0,0 +1,561 @@
...
+    };
+    const accs = { currentAcc }
+    test(`update CurrentAcc by repayment`, () => {
+        const result = updateCurrAcc("km", "repayment", 1000, "nothing", accs, date, false)
+            // Assertion 
+
+        expect(result).toEqual({
+            owner: 'Test Acc',
+            balance: 9000,
+            movements: addMovement(accs.currentAcc.username, "km", -1000, "repayment", "nothing", date, accs.currentAcc),
```
Comment (sleepingevil) [inline src/updateFunctions.test.js]: There's a better way for doing this, called "mocking". I'll show you next time.

**Comment by sleepingevil** [comment]
Nice work overall. 💯

File: public/index.html (hunk 1 above)
Comment (Barna-Molnar) [reply public/index.html]: Done.

Comment (Barna-Molnar) [reply public/index.html]: But I don't know probably it was an accident ' cos I wasn't aware of this one .Sorry

File: src/Account.jsx (hunk 2 above)
Comment (Barna-Molnar) [reply src/Account.jsx]: Done.

File: src/App.jsx (hunk 3 above)
Comment (Barna-Molnar) [reply src/App.jsx]: It was only a test and after that I forgot to delete the import...

File: src/mixins.scss (hunk 5 above)
Comment (Barna-Molnar) [reply src/mixins.scss:34]: Why I got a thumbs down :D ??

File: src/updateFunctions.test.js (hunk 7 above)
Comment (Barna-Molnar) [reply src/updateFunctions.test.js]: I am really looking forward to see it 👯

**Review by Barna-Molnar — COMMENTED** [review]
Thank you to reviewing my code , I am pleased to hear that I did a good job , and I am going to thinking about changing the "update" words in every test!

File: src/Account.jsx [hunk 11]
```diff
@@ -23,7 +22,7 @@ class Account extends React.Component {
       repayAmount: "",
       repayMessage: "",
       repayRecepient: "",
-      time: 3000,
+      logooutTime: 3000, // it is related to to logoutTimer()
```
Comment (sleepingevil) [inline src/Account.jsx]: camelCase please

File: src/Login.jsx [hunk 12]
```diff
@@ -15,15 +15,7 @@ class Login extends Component {
...
-      <div
-        className={`login-page`}
-        // style={{
-        //   left: this.props.loginVisibility
-        //     ? "calc(100%/2 - 180px)"
-        //     : "calc(100%/2 - 1800px)",
-        //   opacity: this.props.loginVisibility ? 1 : 0,
-        // }}
-      >
+      <div className={`login-page`}>
```
Comment (sleepingevil) [inline src/Login.jsx]: When it's a single string you should pass it like this: 
`className="login-page"`

**Comment by sleepingevil** [comment]
> You should not commit .eslintcache and package-lock.json

These are still in git, you just didn't commit the changes. You should remove them from git.

File: src/mixins.scss (hunk 5 above)
Comment (sleepingevil) [reply src/mixins.scss:34]: Do you need that comment for something?
